tokens/usecase/tokens_usecase.go

```diff
@@ -246,8 +253,18 @@ func (t *tokensUseCase) getPricesForBaseDenom(ctx context.Context, baseDenom str
 		wg.Add(1)
 		go func(baseDenom, quoteDenom string) {
 			defer wg.Done()
-
-			price, err := pricingStrategy.GetPrice(ctx, baseDenom, quoteDenom, pricingOptions...)
+			var price osmomath.BigDec
+			var err error
+			price, err = pricingStrategy.GetPrice(ctx, baseDenom, quoteDenom, pricingOptions...)
```

Comment:
We should also add a unit test for `tokensUseCase.GetPrices(...)` where we specify coingecko pricing option (not fallback but directly) and confirm the right pricing source being called via mocks